package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "patternfly-webcomponents",
-  "version": "0.0.7",
+  "version": "0.0.8",
```

Comment:
If we want to use the release automation, we don't need to bump the version number here.